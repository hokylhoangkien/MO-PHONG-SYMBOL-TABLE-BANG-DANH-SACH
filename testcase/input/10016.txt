ASSIGN _o 8