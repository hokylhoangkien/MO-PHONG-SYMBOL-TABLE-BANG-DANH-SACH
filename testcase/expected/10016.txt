Invalid: ASSIGN _o 8